flows/definition/migrations/specdata/templates.json

```diff
@@ -0,0 +1,91 @@
+{
+    "13.2.0": {
+        "actions": {
+            "add_contact_groups": [
+                ".groups[*].name_match"
+            ],
+            "add_contact_urn": [
+                ".path"
+            ],
+            "add_input_labels": [
+                ".labels[*].name_match"
+            ],
+            "call_classifier": [
+                ".input"
+            ],
+            "call_resthook": [],
+            "call_webhook": [
+                ".body",
+                ".headers.*",
+                ".url"
+            ],
+            "enter_flow": [],
+            "open_ticket": [
+                ".assignee.email_match",
+                ".body"
+            ],
+            "play_audio": [
+                ".audio_url"
+            ],
+            "remove_contact_groups": [
+                ".groups[*].name_match"
+            ],
+            "request_optin": [],
+            "say_msg": [
+                ".text"
+            ],
+            "send_broadcast": [
+                ".attachments[*]",
+                ".contact_query",
+                ".groups[*].name_match",
+                ".legacy_vars[*]",
+                ".quick_replies[*]",
+                ".text"
+            ],
+            "send_email": [
+                ".addresses[*]",
+                ".body",
+                ".subject"
+            ],
+            "send_msg": [
+                ".attachments[*]",
+                ".quick_replies[*]",
+                ".templating.variables[*]",
```

Comment:
this is missing template params... and I'm struggling to make that work and starting to wonder about a different format